src/components/bs5/navbar/navbar.data.json

```diff
@@ -141,6 +141,126 @@
                 "icon": "",
                 "target_url": "index.html"
             }
+        ],
+        "CTA": [
```

Comment:
CTA's for mobile view - appear the same as standard nav links despite design (I think the design is wrong)

@stvp-qld